src/controllers/profile.js

```diff
@@ -0,0 +1,75 @@
+import profileImage from '../middlewares/profile';
+import UserService from '../services/user.service';
+import successResponse from '../utils/successResponse';
+import ErrorResponse from '../utils/errorResponse';
+
+const { findUser, updateUser } = UserService;
+
+class Profile {
+  static async getProfile(req, res, next) {
+    const { id } = req.params;
+    try {
+      const user = await findUser({ id });
+
+      return successResponse(res, 200, `Successfully retrieved profile`, user);
+    } catch (error) {
+      if (error.message === 'User not found') {
+        return ErrorResponse.notFoundError(
+          res,
+          404,
+          `Ooops! No such User was found`
+        );
+      }
+      return next(
+        new ErrorResponse.internalServerError(
+          res,
+          500,
+          `Error occured while fetching your profile ${error.message}`
+        )
+      );
+    }
+  }
+
+  static async updateProfile(req, res, next) {
+    const { id } = req.params;
+    try {
+      const user = await findUser({ id });
+
```

Comment:
This way is wrong.
You have the id in the token and a user to complete the profile must be 
authenticated first. no need to add an unnecessary param (**id**)